src/content/en/2019/performance.md

```diff
@@ -30,232 +30,177 @@ Almost all of the other chapters in the Web Almanac are based on data from the [
-</figure>
-
-In Figures 2 and 3, the results show that only 2.17% of websites are considered fast while 60.28% of websites are considered slow. To help us understand how users experience FCP across different devices, let's segment by form factor.
+To help us understand how users experience FCP across different devices, let's segment by form factor.
 
 #### FCP by device
 
-<figure>
-// Chart: Flame distribution of 07_01b
-<figcaption>Figure 4. Distribution of _desktop_ websites' fast, average, and slow FCP performance.</figcaption>
+<figure id="fig3">
+<iframe width="600" height="371" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSQlf3_ySLPB5322aTumUZhbVGdaUdkmi1Hs4bYuO3Z1kqM4xspx7REbwXukwPd_tsOSg6oImzpYLM9/pubchart?oid=1231764008&amp;format=interactive"></iframe>
+<figcaption markdown>Figure 3. Distribution of <em>desktop</em> websites' fast, moderate, and slow FCP performance.</figcaption>
 </figure>
 
-<figure>
-// Chart: Flame distribution of 07_01c
-<figcaption>Figure 5. Distribution of _phone_ websites' fast, average, and slow FCP performance.</figcaption>
+<figure id="fig4">
+<iframe width="600" height="371" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSQlf3_ySLPB5322aTumUZhbVGdaUdkmi1Hs4bYuO3Z1kqM4xspx7REbwXukwPd_tsOSg6oImzpYLM9/pubchart?oid=167423587&amp;format=interactive"></iframe>
+<figcaption markdown>Figure 4. Distribution of <em>phone</em> websites' fast, moderate, and slow FCP performance.</figcaption>
 </figure>
 
-In Figures 4 and 5 above, the FCP distributions of 1,000 sample websites are broken down by desktop and phone. It's subtle, but the torso of the desktop fast FCP distribution appears to be more convex than the distribution for phone users. This visual approximation suggests that desktop users experience a higher overall proportion of fast FCP. To verify this we can apply the PSI methodology to each distribution.
+In Figures 3 and 4 above, the FCP distributions are broken down by desktop and phone. It's subtle, but the torso of the desktop fast FCP distribution appears to be more convex than the distribution for phone users. This visual approximation suggests that desktop users experience a higher overall proportion of fast FCP. To verify this we can apply the PSI methodology to each distribution.
 
-<figure>
-// Chart: Bar distributions of 07_03b
-<figcaption>Figure 6. Distribution of websites labelled as having fast, average, or slow FCP, broken down by device type.</figcaption>
+<figure id="fig5">
+<iframe width="600" height="371" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSQlf3_ySLPB5322aTumUZhbVGdaUdkmi1Hs4bYuO3Z1kqM4xspx7REbwXukwPd_tsOSg6oImzpYLM9/pubchart?oid=486448175&amp;format=interactive"></iframe>
+<figcaption>Figure 5. Distribution of websites labelled as having fast, moderate, or slow FCP, broken down by device type.</figcaption>
```

Comment:
How should we tackle adding descriptive alt text to all the charts? In a separate PR after charts have been added and copy-editing is complete?

Reply:
The figures should all contain a figcaption where we describe the contents of the figure. Then in the paragraphs before/after the figure we should be describing what the key takeaways are from the chart. So even if we don't convey all of the detail of the visualization ("desktop Foo is X%, mobile Foo is Y%, desktop Bar is Z%, etc") we're still getting the point across in text so that it's understandable to all readers.

How does that sound?

Reply:
I don't think every chapter has been written in a way that spells out what each of the values in the charts are. We might have to do a lot of editing...

Think the best solution may be to have a short figcaption like you are describing, but also have additional [screen reader only](https://webaim.org/techniques/css/invisiblecontent/) content in the figcaptions as well.

If we do this, I could put together some templates to follow so we get descriptions like below... but it still might be tough to get all this text written by the deadline...
```
<insert graphic of metric 09_26>

*Figure 2. Percentage of sites that disable zooming and scaling vs device type*

Alt text: Figure 2. Vertical grouped bar chart titled "Are zooming and scaling enabled?" measuring percentage data, ranging from 0 to 80 in increments of 20, vs. the device type, grouped into desktop and mobile. Desktop enabled: 75.46%; Desktop disabled 24.54%; Mobile enabled: 67.79%; Mobile disabled: 32.21%.
```

Reply:
Yeah, having longer descriptions in off-screen figcaptions seems like a reasonable approach. I agree it would be tough to get that for all chapters by launch.